cubed/tests/test_array_api.py

```diff
@@ -563,6 +563,74 @@ def test_sum_axis_0(spec, executor):
     assert_array_equal(b.compute(executor=executor), np.array([12, 15, 18]))
 
 
+def test_var(spec, executor):
```

Comment:
Can you parameterize all these tests for `var` for different axes, and for `keepdims`? There are other tests in the test suite that do this.